lib/solidus_importer/process_import.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+module SolidusImporter
+  ##
+  # This class parse the source file and create the rows (scan). Then it asks to
+  # Process Row to process each one.
+  class ProcessImport
+    def initialize(import)
+      @import = import
+      @importer = ::SolidusImporter::Config.solidus_importer.dig(@import.import_type.to_sym, :importer)
+      validate_context
```

Comment:
validate_context here doesn't seem to actually validate the context, but rather, to validate the ProcessImport instance, what about naming it `validate!` (the bang would signal it's gonna raise an exception if it's not valid)